vocode/streaming/agent/state_agent.py

```diff
@@ -392,9 +396,10 @@ def __init__(
         else:
             self.base_url = getenv("AI_API_BASE")
             self.model = getenv("AI_MODEL_NAME_LARGE")
+        key = getenv("AI_API_KEY")
```

Comment:
i don't think we need this part. Just get the model from the config, like this `self.model = self.agent_config.model`